Bot.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+
+import telebot
+import ORMForBot as ORM
+from telebot import types
+import TicTacToeForBot as TicTacToe
```

Comment:
I suppose it's better to regroup imports – at first, all `import ...`, then all `from ... import ...`

And to add 2 blank lines after all your imports

Reply:
Done